mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule Riemannx.Mixfile do
   use Mix.Project
 
-  @version "4.1.3"
+  @version "4.1.4"
```

Comment:
You shouldn't bump this, I have to do it as part of the git flow operation.

Reply:
I will remove